cpu/esp32/ld/esp32.common.ld

```diff
@@ -41,6 +41,20 @@ SECTIONS
     _rtc_data_end = ABSOLUTE(.);
   } > rtc_slow_seg
 
+  /* This section holds data that should not be initialized at power up
+     and will be retained during deep sleep. The section located in
+     RTC SLOW Memory area. User data marked with RTC_NOINIT_ATTR will be placed
+     into this section. See the file "esp_attr.h" for more information.
+  */
+  .rtc.noinit (NOLOAD):
+  {
+    . = ALIGN(4);
+    _rtc_noinit_start = ABSOLUTE(.);
+    *(.rtc_noinit .rtc_noinit.*)
+    . = ALIGN(4) ;
+    _rtc_noinit_end = ABSOLUTE(.);
+  } > rtc_slow_seg
```

Comment:
Over [at](https://github.com/RIOT-OS/RIOT/blob/master/cpu/lpc2387/ldscripts/lpc2387.ld#L188) [ARM](https://github.com/RIOT-OS/RIOT/blob/master/cpu/cortexm_common/ldscripts/cortexm_base.ld#L173) we call this section `.backup`, but I see you already have `.rtc.data` et al. 

So it's really time for some common macros to wrap those - `__attribute__((section(".backup.bss")))` is neither very easy to type nor visually appealing anyway.

Reply:
> Over [at](https://github.com/RIOT-OS/RIOT/blob/master/cpu/lpc2387/ldscripts/lpc2387.ld#L188) [ARM](https://github.com/RIOT-OS/RIOT/blob/master/cpu/cortexm_common/ldscripts/cortexm_base.ld#L173) we call this section `.backup`, but I see you already have `.rtc.data` et al.

Yes, I know. Indeed, I alreay had a [branch](https://github.com/gschorcht/RIOT-Xtensa-ESP/tree/cpu/esp32/backup_ram) which defines `.backup.bss` and `.backup.data` sections in RTC memory to be able to use `tests/periph_backup_ram` together with PR #13365. However, I didn't provide it as PR, because the RTC memory it is not a backup RAM in strong sense. The RTC memory cannot be separately powered by a backup battery.

Reply:
> I didn't provide it as PR, because the RTC memory it is not a backup RAM in strong sense. The RTC memory cannot be separately powered by a backup battery.

I wouldn't say this is a strong requirement.
The idea is that this memory is still retained when all other memory is lost, so from a software perspective everything else is irrelevant.